src/main/scala/mesosphere/marathon/core/matcher/manager/impl/OfferMatcherManagerActor.scala

```diff
@@ -164,7 +164,7 @@ private[impl] class OfferMatcherManagerActor private (
   }
 
   def offersWanted: Boolean = matchers.nonEmpty && launchTokens > 0
-  def updateOffersWanted(): Unit = offersWantedObserver.onNext(offersWanted)
+  def updateOffersWanted(): Unit = offersWantedInput.offer(offersWanted)
```

Comment:
My opinion is no. The data we are transmitting are booleans and I see no possible way that it will back up. Setting the buffer mode to back pressure is effectively the same as an actor tell - unbounded queue in the end.